src/assets/theme.scss

```diff
@@ -443,3 +443,8 @@ input {
 .capitalize {
   text-transform: capitalize;
 }
+
+// ------ Modal text lable alignment ------ //
+.modal-text-align {
+  margin-top: 20px !important;
```

Comment:
This doesn't belong into `theme.scss`! Keep this file clean. If you want to add something that is shared across modals, there's a `app/modals/shared/_modal-layout.scss` instead.

Reply:
actually this class is using in two components so I thought we can make it in once place

Reply:
I got it (y)

Reply:
Yes, that's why `_modal-layout.scss` is included in all modals.